src/lib/utilities/Dialog/examples/DialogExampleForm.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+	// Props
+	/** Exposes parent props to this component. */
+	export let parent: any;
+
+	// Stores
+	import { dialogStore } from '$lib/utilities/Dialog/stores';
+
+	// Form Data
+	const formData: any = {
+		name: 'Jane Doe',
+		tel: '214-555-1234',
+		email: 'jdoe@email.com'
+	};
+
+	// We've created a custom submit function pass the response and close the dialog.
```

Comment:
Irrelevant typo:
```
// We've created a custom submit function to pass the response and close the dialog.
```

Reply:
The code is being reference for documentation purposes, so very relevant! Good catch!